README.md

```diff
@@ -91,7 +88,7 @@ Find **Protocol Buffers Descriptions** at the [`./pb` directory](./pb).
 
 > 💡  Recommended for demos and making it available publicly.
 
-1. Install tools specified in the previous section (Docker, kubectl, skaffold)
+1. Install tools specified in the previous section (kubectl, skaffold)
```

Comment:
This part of the diff is not accurate: Skaffold will still need a working `docker build` locally even if it is deploying to a remote cluster.

I'm tempted to think: for now Docker for Desktop is doing a decent job here and we should probably keep it.

Reply:
@ahmetb It's fine with keeping DfD, the primary reason why I've added minikube - it's available for Linux.